react-vite/package.json

```diff
@@ -5,7 +5,7 @@
   "type": "module",
   "scripts": {
     "dev": "vite",
-    "build": "CI=false && vite build",
+    "build": "CI=false && vite build --watch",
```

Comment:
Trying to keep the --watch off of the main branch, it will cause errors when deployed